### PR TITLE
Introducing use of three Metazoa Protein Tree Reference sets 

### PR DESCRIPTION
**Pull request by ens-LCampbell**
## Description

This PR introduces the use of multiple [x3] protein trees for main site compara computation of metazoan invertebrate homologies. The three sets are: 
(1) Insect only cluster (Main cluster allocation of maximum 125 species).
(2) protostome invertebrates, with a selection of arthropod species (Secondary cluster, with a maximum sp allocation of 75) species.
(3) default Metazoa wide (excluding vertebrates) reference set encompassing all currently hosted metazoan phyla (Secondary cluster, with a maximum sp allocation of 75) species. 

**Related JIRA tickets:**
- [ENSMETAZOA-42](https://www.ebi.ac.uk/panda/jira/browse/ENSMETAZOA-42)

## Overview of changes
Given the nature of these protein tree reference sets and the in-group taxa required, some species overlap was unavoidable. To counter this, select clades have been excluded and specific species reintroduced in order to isolate in-group taxa to a defined set of species for each set. 

## Testing
Not tested, entering uncharted waters.... 

## Notes
Aim is to implement only one cluster at a time:
e110: maintain the current protein tree (unchanged) and compute the large "Main" cluster
e111: retire current protein tree and compute the other two (smaller) clusters

### CODE REVIEW

File: conf/metazoa/allowed_species.json [hunk 1]
```diff
@@ -163,13 +185,15 @@
...
     "tribolium_castaneum",
     "trichinella_spiralis",
     "trichogramma_pretiosum_gca000599845v3",
     "trichoplax_adhaerens",
     "trichuris_muris",
-    "varroa_destructor_gca002443255",
-    "zootermopsis_nevadensis"
+    "venturia_canescens_gca019457755v1rs",
+    "zerene_cesonia_gca012273895v2rs",
+    "zootermopsis_nevadensis",
```
Comment (JAlvarezJarreta) [inline conf/metazoa/allowed_species.json:198]:
```suggestion
    "zootermopsis_nevadensis"
```
I'm afraid Travis is complaining about this one (malformed JSON).

Comment (ens-LCampbell) [reply conf/metazoa/allowed_species.json]: Ooops, nice catch ! My mistake, thank you.

**Review by CristiGuijarro — APPROVED** [review]

**Review by JAlvarezJarreta — COMMENTED** [review]
Travis is failing because it is checking the `allowed_species.json` file **_before_** the changes proposed. I have double-checked and the errors raised have been successfully corrected by @ens-LCampbell already.

**Comment by twalsh-ebi** [comment]
> Travis is failing because it is checking the `allowed_species.json` file **_before_** the changes proposed. I have double-checked and the errors raised have been successfully corrected by @ens-LCampbell.

Is this definitely the case for all of them, @JAlvarezJarreta? For example, it looks like `acyrthosiphon_pisum` is renamed to `acyrthosiphon_pisum_gca005508785v2` in the `allowed_species.json` of this PR but not in the `mlss_conf.xml`.

**Comment by JAlvarezJarreta** [comment]
@twalsh-ebi that is how vertebrates are defined, so we followed the same approach. Are you telling me you are planning to rewrite all vertebrates due to the same lack of review? Once this is set, the agreed cap can easily be controlled via the `allowed_species.json` file.

**Comment by JAlvarezJarreta** [comment]
> > Travis is failing because it is checking the `allowed_species.json` file **_before_** the changes proposed. I have double-checked and the errors raised have been successfully corrected by @ens-LCampbell.
> 
> Is this definitely the case for all of them, @JAlvarezJarreta? For example, it looks like `acyrthosiphon_pisum` is renamed to `acyrthosiphon_pisum_gca005508785v2` in the `allowed_species.json` of this PR but not in the `mlss_conf.xml`.

My apologies @twalsh-ebi, the error that Travis was showing when I checked was different than this one. And this one needs to be resolved from our side indeed :+1:

**Comment by ens-LCampbell** [comment]
@twalsh-ebi Hi Thomas following your request I have updated the mlss conf to now only reflect said protein sets in terms of in species 'genome' elements only. 238 species (197 unique species) total across all 3 sets with some overlap in species. 

We still have not reached the max sp limit for each of the sets. 108 sp for the large cluster (Insects, max 125 sp), then 60 and 70 (secondary clusters, max 75 sp each) for protostome and metazoa clusters respectively. 

Do let me know if there are any other concerns. Note that we are still working loading some of the included species for release e110 so we will be keeping an eye on these species.

**Review by ens-sb — APPROVED** [review]

**Review by twalsh-ebi — APPROVED** [review]

**Pull request merged**